examples/llava/clip.cpp

```diff
@@ -2651,6 +2735,13 @@ bool clip_image_batch_encode(clip_ctx * ctx, const int n_threads, const clip_ima
     // copy the embeddings to the location passed by the user
     ggml_backend_tensor_get(embeddings, vec, 0, ggml_nbytes(embeddings));
 
+    if(ctx->has_glm_projector){
+        //eoi
+        ggml_tensor * eoi = ctx->vision_model.eoi_w;
+        int offset=ggml_nelements(eoi)*clip_n_patches(ctx);
```

Comment:
Maybe this is more straight-forward to understand?

```suggestion
        int offset = ggml_nelements(embeddings);
```